db/schema.rb

```diff
@@ -10,818 +10,821 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_06_03_002652) do
+ActiveRecord::Schema.define(version: 2022_06_20_054530) do
```

Comment:
@developedBySJ The entire file has changed. Please do check if your Prettier is setup correctly.